addons/point_of_sale/static/src/xml/pos.xml

```diff
@@ -1759,7 +1759,7 @@
                 <div><div t-attf-style="background-image:url(#{orderline.product.image_base64})"/></div>
                     <div><t t-esc="orderline.product.display_name" /></div>
                     <div><t t-esc="orderline.get_quantity_str()" /></div>
-                    <div><t t-esc="widget.format_currency(orderline.get_display_price())"/></div>
+                    <div><t t-esc="widget.pos.format_currency(orderline.get_display_price())"/></div>
```

Comment:
why did you do this modifications as "format_currency" is implemented on base widget?

Reply:
Because `widget` is not what you think it is 😁 : it's an object containing only 1 property `pos` instead of the `chrome` instance (see https://github.com/odoo/odoo/pull/36786/files/76a032152364dd74a8bce2b4375049b1560b2a16#diff-1c14accbe350f7a177b3e81e5846861bR820). 

The idea is to reduce the cross-reference to the `chrome` instance in the model (as this template is rendered manually from the model).